src/app/register/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { getProviders } from "next-auth/react";
+
+import { Seo } from "../../components/common/Seo";
+import { RegisterView } from "../../views/register/Register";
+
+const Register = async () => {
+  const providers = await getProviders();
+
+  if (!providers) {
+    return null;
+  }
+
+  return (
+    <>
+      <Seo />
+      <RegisterView providers={providers} />
+    </>
+  );
+};
+
+export default Register;
```

Comment:
I guess that you didn't pull the newest changes from the current branch? Because as I remember these changes were introduced in the previous PR?